.cbmc-batch/stubs/aws_byte_buf_write_stub.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <aws/common/array_list.h>
+#include <aws/common/byte_order.h>
+#include <aws/common/common.h>
+
+#include <string.h>
+
+/* We can't include byte_buf.h, because that defines this as a static inline function, and we need to override it.
```

Comment:
Why is this comment relevant?  You are using the static removal flag, resulting in the name mangling below.  Can't you include the header file defining the unmangled name so that you don't have to copy the struct definitions (a source of future error)?

Reply:
No longer needed after refactoring the headers.